components/arm/fake/fake.go

```diff
@@ -100,8 +103,8 @@ func NewArm(cfg config.Component, logger golog.Logger) (arm.LocalArm, error) {
 	case modelPath != "":
 		model, err = referenceframe.ModelFromPath(modelPath, cfg.Name)
 	default:
-		// if no arm model is specified, we return empty one with 0 dof and 0 spatial transformation
-		model = referenceframe.NewSimpleModel(cfg.Name)
+		// if no arm model is specified, we return the ur5e
+		model, err = referenceframe.ParseModelJSONFile(utils.ResolveFile("components/arm/universalrobots/ur5e.json"), cfg.Name)
```

Comment:
We continue to not want to do this.

Reply:
I don't see why the default fake arm should not have more DoF, whether done this way or differently. this is more usable and allows users/testers to actually do stuff with the fake arm without any configuration. Users can always specify a different model json if they want